### DISCUSSION TITLE
为什么要把敬爱的毛主席讲的话删掉？

### DISCUSSION THREAD

**Discussion by syd951186545** [WeNeedHome/SummaryOfLoanSuspension#522] [General]
为什么要把敬爱的毛主席讲的话删掉？

**Comment by hktk1024** [comment]
别的不说 58-60年你查查人口去哪里了

**Comment by duhanmin** [comment]
将事实,没必要扯虎皮拉大旗

**Comment by anamsuan** [comment]
没感觉哪里需要敬爱

**Comment by volkerzht** [comment]


**Reply by zcaaron** [reply]
饿死了

**Reply by lkr4545plb** [reply]
在那看的地摊文学在这乱BB，气象数据你倒是贴出来啊 ，几千万人的数据你倒是贴出来啊

**Comment by luuthink** [comment]
毛主席可是全世界干货最多的人，他说的话肯定不会错的

**Comment by allenyan513** [comment]
说明变正常了

**Comment by dongyc519** [comment]
这种引战话题可以停止了

**Comment by go2006010215** [comment]
1，延安整风运动
整风运动的实行，使毛泽东成为党主席，在党中央的地位更为确立，被许多学者认为是毛泽东个人崇拜的开端。 该运动造成诸多冤假错案，大量人员遭到迫害，知识分子群体受到巨大冲击。 有研究指该运动，共造成超过一万人死亡。

来源
https://zh.m.wikipedia.org/zh-hans/%E6%95%B4%E9%A2%A8%E9%81%8B%E5%8B%95

2，反AB团
朱家浩承認红旗社列宁青年社、赣西南政府都有AB团的小组，赣西南AB团的总团长是谢兆元。 於是赣西南特委抓捕谢兆元等人。 當年5月至9月，于都有1000余人被指稱為AB团分子並遭到捕杀，城关区就有近200人被杀。

来源
https://zh.m.wikipedia.org/zh-hans/%E5%8F%8DAB%E5%9C%98

三年困难时期1500万至5500万人的非正常死亡
据各方估计，三年困难时期间发生的大饥荒共造成了中国大陆1500万至5500万人的非正常死亡，被广泛视为人类历史上规模最大的饥荒，也被认为是人类历史上最严重的人为灾难之一

来源
https://zh.wikipedia.org/wiki/%E4%B8%89%E5%B9%B4%E5%9B%B0%E9%9A%BE%E6%97%B6%E6%9C%9F

袁隆平：三年饥荒饿死几千万
https://www.youtube.com/watch?v=EM3vH73tiSg

4，反右运动

来源
https://zh.wikipedia.org/wiki/%E5%8F%8D%E5%8F%B3%E8%BF%90%E5%8A%A8

文革期间非正常死亡人口达数十万至2000万不等
据各方估计，文革期间中国大陆非正常死亡人口达数十万至2000万不等，从北京的“红八月”起，诸多地区发生了大屠杀，包括广西文革屠杀（及大规模人吃人事件）、内蒙古内人党事件、广东文革屠杀、云南文革屠杀、湖南文革屠杀等，亦爆发了二十世纪世界上最重大的人为技术灾难“河南“75·8”水库溃坝”。
来源
https://zh.wikipedia.org/wiki/%E6%96%87%E5%8C%96%E5%A4%A7%E9%9D%A9%E5%91%BD
统计：

林辉：中共70年来杀了多少人？
https://www.epochtimes.com/gb/19/9/25/n11546701.htm
